core-services/pdc/src/main/java/org/silverpeas/core/pdc/pdc/service/DefaultPdcClassifyManager.java

```diff
@@ -83,9 +79,6 @@ public int addPosition(int silverObjectId, ClassifyPosition position, String sCo
       connection = DBUtil.openConnection();
       // Vérification de la contrainte invariante
       int nPositionId = classifyEngine.classifySilverObject(connection, silverObjectId, position);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "nPositionId" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1481)